nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-api/src/main/java/org/nuxeo/ecm/platform/audit/api/AuditReader.java

```diff
@@ -47,17 +66,67 @@
      * @deprecated since 8.4, use {@link #getLogEntriesFor(String, String))} instead.
      */
     @Deprecated
-    List<LogEntry> getLogEntriesFor(String uuid);
+    default List<LogEntry> getLogEntriesFor(String uuid) {
+        return queryLogs(
+                new AuditQueryBuilder().filters(AuditFilterBuilders.eq(LOG_DOC_UUID, uuid))
+                                       .defaultOrder());
+    }
 
     /**
      * Returns the logs given a doc uuid, a map of filters and a default sort.
      *
      * @param uuid the document uuid
      * @param filterMap the map of filters to apply
-     * @param doDefaultSort the default sort to set
+     * @param doDefaultSort the default sort to set (eventDate desc)
      * @return a list of log entries
+     * @deprecated since 9.3, this method doesn't take into account the document repository, use
+     *             {@link #getLogEntriesFor(String, String, Map, boolean)} instead.
      */
-    List<LogEntry> getLogEntriesFor(String uuid, Map<String, FilterMapEntry> filterMap, boolean doDefaultSort);
+    @Deprecated
+    default List<LogEntry> getLogEntriesFor(String uuid, Map<String, FilterMapEntry> filterMap, boolean doDefaultSort) {
+        // create filters
+        List<AuditFilterBuilder> filters = new ArrayList<>();
+        filters.add(AuditFilterBuilders.eq(LOG_DOC_UUID, uuid));
+        filterMap.values()
+                 .stream()
+                 .map(fme -> new AuditFilterBuilder(fme.getColumnName(), fme.getOperator(), fme.getObject()))
+                 .forEach(filters::add);
+        // create builder
+        AuditQueryBuilder builder = new AuditQueryBuilder().filters(filters);
+        if (doDefaultSort) {
+            builder.defaultOrder();
+        }
+        return queryLogs(builder);
+    }
```

Comment:
Can you move these long default methods to `AbstractAuditBackend` instead? I don't find it nice to have such amount of code in interfaces.